README.md

```diff
@@ -1,2 +1,19 @@
 # nickel-nix
-An experimental Nix toolkit to use nickel as a language for writing nix packages, shells and more.
+
+An experimental Nix toolkit to use nickel as a language for writing nix
+packages, shells and more.
+
```

Comment:
We need to mention that only equivalent to `nix-shell` is provided, but more is to come. This is to manage expectations when somebody comes to here.